pom.xml

```diff
@@ -12,24 +12,28 @@
     <parent>
         <groupId>uk.gov.companieshouse</groupId>
         <artifactId>companies-house-parent</artifactId>
-        <version>1.3.0</version>
+        <version>1.3.3-rc1</version>
     </parent>
 
     <properties>
         <java.version>11</java.version>
-        <spring-boot-dependencies.version>2.5.6</spring-boot-dependencies.version>
-        <spring-boot-maven-plugin.version>2.5.6</spring-boot-maven-plugin.version>
+<!--        <spring-boot-dependencies.version>2.5.6</spring-boot-dependencies.version>-->
+        <spring-boot-dependencies.version>2.7.1</spring-boot-dependencies.version>
+<!--        <spring-boot-maven-plugin.version>2.5.6</spring-boot-maven-plugin.version>-->
+        <spring-boot-maven-plugin.version>2.7.1</spring-boot-maven-plugin.version>
         <rest-service-common-library.version>0.0.5</rest-service-common-library.version>
         <maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
         <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
-        <ch-kafka.version>1.4.5</ch-kafka.version>
+        <ch-kafka.version>1.4.8</ch-kafka.version>
         <structured-logging.version>1.9.14</structured-logging.version>
         <junit-jupiter.version>5.5.2</junit-jupiter.version>
```

Comment:
I have tried bumping
junit-jupiter to 5.8.2
and
mockito to 3.12.4
without ill effect (the tests still pass). So that might be worth a try.